docs/en/quantization/w8a8.md

```diff
@@ -1,55 +1,62 @@
 # SmoothQuant
 
-LMDeploy provides functions for quantization and inference of large language models using 8-bit integers.
+LMDeploy provides functions for quantization and inference of large language models using 8-bit integers(INT8). For GPUs such as Nvidia H100, lmdeploy also supports 8-bit floating point(FP8).
```

Comment:
May list the supported cuda arch for each quantization datatype like `w4a16.md` did

Reply:
ok